DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces/component_interfaces_comparison/pipes/Readme.md

```diff
@@ -0,0 +1,184 @@
+# Streaming Data Interfaces (Pipes)
+This implementation uses a register-mapped invocation interface, and demonstrates how to use pipes to configure a streaming data interface.
+
+![](../assets/agent_n_pipe.svg)
+
+## Invocation Interface
+By default, an un-decorated oneAPI kernel will have all its control signals and arguments mapped into the IP component's control/status register (CSR).
+
+## Data Interface - Streaming (Pipes)
+Pipes are a first-in first-out (FIFO) buffer construct that provide links between elements of a design. Pipes that connect a host and a device are referred to as host pipes. In this design, interfaces `InputPipeA`, `InputPipeB`, `OutputPipeC` are implemented as host pipes.
```

Comment:
Should we still be referring to "host pipes" here,  given that we've removed the name elsewhere? perhaps we can say something along the lines of:

Pipes are a first-in first-out (FIFO) buffer construct that provide links between elements of a design. In this design, we use pipes to implement the inputs and outputs as streaming data interfaces.